notification/twilio/twiml.go

```diff
@@ -48,6 +48,7 @@ const (
 	optionCancel
 	optionConfirmStop
 	optionAck
+	optionEsc
```

Comment:
nit: let's spell this out since `esc` is a little ambiguous without context
```suggestion
	optionEscalate
```